### DISCUSSION TITLE
Improve usability by adding broader GPU support

### DISCUSSION THREAD

**Discussion by marcin-przywoski** [hayabhay/whisper-ui#32] [Ideas]
Hi, please check out this repo : 

https://github.com/Const-me/Whisper

This guys created a CLI for Whisper which also has the ability to use GPU for the work (works also on AMD not only on Nvidia!), it's also way faster

**Comment by hayabhay** [comment]
Ah! Will definitely look into it. It does sound like there is a pattern of CPU-only & broad GPU support needed.

Will try to design the app in a way that is decoupled from the underlying whisper implementation for the next version.

**Comment by dixan51** [comment]
> Hi, please check out this repo :
> 
> https://github.com/Const-me/Whisper
> 
> This guys created a CLI for Whisper which also has the ability to use GPU for the work (works also on AMD not only on Nvidia!), it's also way faster

Finally, whisper-ui whitout GPU parameter is totally useless.

**Reply by marcin-przywoski** [reply]
But the best thing is that it's faster that the original implementation and also works on all cards! Before I had to install Linux (and take my time to install custom ROCm installation to make it work on AMD) to use GPU with it, now I can do that in Windows and it's also waaaay faster!
